src/renderer/components/TeamDropdownButton.tsx

```diff
@@ -65,7 +65,7 @@ const TeamDropdownButton: React.FC<Props> = (props: Props) => {
                 <i className='icon-server-variant'/>
                 {badgeDiv}
             </div>
-            <span>{activeServerName}</span>
+            <span>{activeServerName || 'No servers found'}</span>
```

Comment:
nit, maybe there is even better wording for that
```suggestion
            <span>{activeServerName || 'No servers configured'}</span>
```